module/Activity/Module.php

```diff
@@ -52,12 +54,27 @@ public function getServiceConfig()
                 'activity_doctrine_em' => function ($sm) {
                     return $sm->get('doctrine.entitymanager.orm_default');
                 },
+                'activity_form_activity' => function ($sm) {
+                    /** @var \Decision\Service\Member $userService */
+                    $memberService = $sm->get('decision_service_member');
+                    /** @var \User\Model\User $identity */
+                    $identity = $sm->get('user_role');
+
+                    // This is a necessary hack, since the user and corresponding member model is cached
+                    // I could not get the organ from the Member, because this resulted in all kinds of exceptions
+                    // This forces doctrine to fetch a clean object from the database
+                    $user = $memberService->findMemberByLidNr($identity->getLidnr());
+
```

Comment:
Perhaps it is easier to just add a function (if it doesn't exist already) to the member mapper which returns all organs which a member is currently part of. This will save some queries, since you don't need the entire member object.